thirdparty/openapi3/components/schemas/ConsentsPostRequest.yaml

```diff
@@ -15,12 +15,12 @@ properties:
     description: |
       The id of the ConsentRequest that was used to initiate the
       creation of this Consent.
-  credential:
-    $ref: ./SignedCredential.yaml
+  
```

Comment:
Why is this removed? I need it for the initial POST consents
![image](https://user-images.githubusercontent.com/5932442/120643254-8a757480-c43b-11eb-86cd-8f849175d10e.png)

Reply:
we have two POST /consents
- the first when DFSP asks PISP to deliver the credential: https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#15-grant-consent
- the second when DFSP ask AUTH-SERVICE to register the consent with credential (as action on PUT /consent/{ID} call made by PISP to DFSP
https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#162-registering-the-credential

in the first one, there is no need for credential prop
in the second, the credential property is mandatory

I think that we need two independent schemas for these two flows, elsewhere there will be a need to implement additional validation logic to check presence/absence of credential field.

Reply:
but defining two independent schemas for the same path and method would be a challenge. 
Which schema should be used is strictly dependent on the receiver context - PISP or AUTH-SERVICE

Reply:
> I think that we need two independent schemas for these two flows, elsewhere there will be a need to implement additional validation logic to check presence/absence of credential field.

Sounds good. I like the multiple independent schemas better.